rdagent/core/evolving_framework.py

```diff
@@ -51,6 +51,13 @@ class EvoStep:
     evolvable_subjects: EvolvableSubjects
     queried_knowledge: QueriedKnowledge | None = None
     feedback: Feedback | None = None
+    code_change_summary: str | None = None  # TODO: minrui
```

Comment:
I think this place is a good place to add summary, 
Let's have the discussion later.